app/uk/gov/hmrc/economiccrimelevyregistration/views/AmlSupervisorView.scala.html

```diff
@@ -37,10 +37,13 @@
     }
 
     <h1 class="govuk-heading-l">@messages("amlSupervisor.heading")</h1>
+
     <p class="govuk-body">@messages("amlSupervisor.p1")</p>
     <p class="govuk-body">@messages("amlSupervisor.p2")</p>
     <p class="govuk-body">@messages("amlSupervisor.p3")</p>
 
+    <h2 class="govuk-heading-m">@messages("amlSupervisor.h2")</h2>
```

Comment:
You can set this as the form label now so that it's not hidden - that's the idea of this pattern is so that you don't have to have hidden form inputs labels